includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -95,11 +95,74 @@ public function getData() {
 		return $data;
 	}
 
-	private function getHref( $hrefKey, $protocolRelative = false ) {
+	protected function getDomainForCityId( $cityId ) {
+		global $wgServer;
+		if ( $cityId === null ) {
+			return $wgServer;
+		} else {
```

Comment:
The `else` isn't needed since it just contains a return, and removing it would be consistent with the other methods like `getDevDomain`.